src/templates/ratings/add.html

```diff
@@ -7,6 +7,11 @@
           {% endif %}>
       {% if isModal %}
         <h3>{{ existingReview and _('Edit Review') or _('Leave a Review') }}</h3>
+	<p class="reviews--guideline">
+          {{ _('Please read the <a href="{url}">Review Guidelines</a> for more details about rating apps.
```

Comment:
can you add `rel="external"` to the anchor? (this will let it open as a new tab/outside of the app.)